COPYRIGHT.txt

```diff
@@ -229,7 +234,7 @@ License: Expat
 
 Files: ./thirdparty/etcpak/
 Comment: etcpak
-Copyright: 2013-2022, Bartosz Taudul
+Copyright: 2013-2024, Bartosz Taudul
```

Comment:
We use the years from their own copyright statement, so as long as they don't update upstream, we shouldn't either:
```
$ grep Copyright thirdparty/etcpak/LICENSE.txt 
Copyright (c) 2013-2022, Bartosz Taudul <wolf@nereid.pl>
```